errors/es-mx.po

```diff
@@ -0,0 +1,1474 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: squid\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2017-09-16 17:53+1300\n"
+"PO-Revision-Date: 2010-08-17 06:57+0200\n"
```

Comment:
This should be the date and time you last changed these texts.

Reply:
Dully noted.